From d7337d4c9f64a4f69e79223899762ad396ede998 Mon Sep 17 00:00:00 2001
From: David Sniatkiwicz <davidsn@marvell.com>
Date: Mon, 2 Jan 2017 10:57:19 +0200
Subject: [PATCH 0708/1345] dts: a80x0: align a8040-db device tree with setup
 A

commit  cd4f6c7f80d6d7d1d36c09a9dad4a3e12150e076 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Update setup A of a8040-db according to board topology.
- cp0: Add i2c
- cp1: Add serial

Change-Id: I9a74508bbbb862b68913a45dec284c247c1e99e6
Signed-off-by: David Sniatkiwicz <davidsn@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35222
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-db-A.dts |    9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-A.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-A.dts
index 0b971ec..1f54584 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-A.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-A.dts
@@ -99,6 +99,12 @@
 			sdhci@780000 {
 				status = "okay";
 			};
+			i2c@701000 {
+				clock-frequency = <100000>;
+				pinctrl-names = "default";
+				pinctrl-0 = <&cpm_i2c0_pins>;
+				status = "okay";
+			};
 			mdio@12a200 {
 				#address-cells = <1>;
 				#size-cells = <0>;
@@ -154,6 +160,9 @@
 			usb3h0_1: usb3@500000 {
 				status = "okay";
 			};
+			serial@702000 {
+				status = "okay";
+			};
 			sata@540000 {
 				status = "okay";
 				sata-port@0 {
-- 
1.7.9.5

